docs/contributing/testing_guide.rst

```diff
@@ -9,6 +9,50 @@ Testing Guide
    :local:
    :backlinks: none
 
+Summary
+=======
+
+* New functionality added to PlasmaPy must also have tests.
+
+* Tests are located in files that begin with :file:`test_` which are
+  inside subdirectories named :file:`tests`.
+
+* Tests are either functions beginning with ``test_`` or classes
+  beginning with ``Test``.
+
+* To install the packages needed to run the tests:
+
+  - Open a terminal.
+
+  - Navigate to the top-level directory (probably named
+    :file:`PlasmaPy`) in your local clone of PlasmaPy's repository.
+
+  - If you are on MacOS or Linux, run:
+
+    .. code-block:: console
+
+       python -m pip install -e.[tests]
```

Comment:
```suggestion
       python -m pip install -e ".[tests]"
```
Missed a space, and the quotations work everywhere, whereas without them it doesn't work on zshell. For some reason.

Reply:
Good catch!  I haven't used zshell, so I wouldn't have realized this on my own.